sphinx/hyperlinks.rst

```diff
@@ -13,7 +13,8 @@
 .. _Cylc Flow: https://github.com/cylc/cylc-flow
 .. _Cylc Rose: https://github.com/cylc/cylc-rose
 .. _Cylc User Guide: https://cylc.github.io/documentation/#the-cylc-user-guide
-.. _Cylc Suite Design Guide: https://cylc.github.io/cylc-doc/current/html/suite-design-guide/suite-design-guide-master.html
+.. _Cylc Suite Design Guide: https://cylc.github.io/cylc-doc/stable/html/suite-design-guide/suite-design-guide-master.html
```

Comment:
At the moment, stable points to 7.x, should this be latest instead for 8.x? Or is the idea to just leave this alone so when Cylc 8 is out proper, it doesn't have to be updated?

Reply:
I couldn't decide which to use so went with stable because at least then we can't forget to update it later.

Reply:
Created an issue - https://github.com/metomi/rose/issues/2457